app/models/claims/claim.rb

```diff
@@ -63,4 +63,12 @@ def submitted_on
   def amount
     Claims::Claim::CalculateAmount.call(claim: self)
   end
+
+  def ready_to_be_checked?
+    mentors.present? && mentor_trainings.without_hours.blank?
+  end
+
+  def remove_mentor_trainings_without_hours
+    mentor_trainings.without_hours.destroy_all
+  end
```

Comment:
This looks harmless enough, but it sets a precedence of data-mutating methods within our models.

Can we move this into a service instead please?